abb_irb1600_support/urdf/irb1600_6_12.urdf

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" ?>
```

Comment:
I don't believe we need to keep providing URDFs anymore. Xacro support is ubiquitous, and URDFs are not composible, so I suggest we remove it from this PR.